settings.php

```diff
@@ -0,0 +1,226 @@
+			'add-css'             => __( 'Add custom CSS to allow showing or hiding the list of sites that use a theme or plugin', 'multisite-enhancements' ),
+			'add-plugin-list'     => __( 'On the network Plugins page, show which blogs have the plugin active', 'multisite-enhancements' ),
+			'add-theme-list'      => __( 'On the network Themes page, show which blogs have the theme active', 'multisite-enhancements' ),
+			'add-site-status'     => __( 'Add status labels for no-index and external domain to blogs in "My Sites" menu', 'multisite-enhancements' ),
+			'add-ssl-identifier'  => __( 'Add an icon to identify the SSL protocol on each site in the network Sites page', 'multisite-enhancements' ),
+			'add-manage-comments' => __( 'Add new "Manage Comments" item with count of comments waiting for moderation in "My Sites" menu', 'multisite-enhancements' ),
+			'add-network-plugins' => __( 'For WordPress earlier than 3.7, add a link to the Plugins page under "Network Admin" in "My Sites" menu', 'multisite-enhancements' ),
+			'add-new-plugin'      => __( 'Enables an "Add New" link under the Plugins menu of each blog, for network admins', 'multisite-enhancements' ),
+			'filtering-themes'    => __( 'Add simple javascript to filter the theme list on network and single site theme page of WordPress backend', 'multisite-enhancements' ),
+			'change-footer'       => __( 'Enhance the admin footer text with RAM, SQL queries and PHP version information', 'multisite-enhancements' ),
+		);
+
+		$options = self::get_settings();
+
+		foreach( $feature_settings as $key => $description ) {
+?>
+			<p>
+				<label>
+					<input type="checkbox" name="wpme_options[<?php echo esc_attr( $key ); ?>]" <?php checked( $options[ $key ], 1 ); ?> value="1">
+					<?php echo $description; ?>
+				</label>
+			</p>
+<?php
+		}
+	}
+
+	/**
+	 * Save / update settings on the database
+	 */
+	public function update_settings() {
+		// check the referer to make sure the data comes from our options page.
+		check_admin_referer( 'wpme_options-options' );
+
+		$options = $_POST['wpme_options'];
+
+		foreach( array_keys( self::$default_options ) as $key ) {
+			if ( ! isset( $options[ $key ] ) )
+				$options[ $key ] = '0';
+		}
+
+		// update option on database
+		update_site_option( 'wpme_options', $options );
+
+		// redirect back to our options page
+		wp_redirect(
+			add_query_arg( array(
+				'page' => 'wpme_config',
+				'updated' => 'true'
+			),
+			network_admin_url( 'settings.php' )
+		));
+		exit;
+	}
+
+	/**
+	 * Load settings from database and merge them with default values
+	 *
+ 	 * @param string [$key] Key of an specific setting desired (optional).
+	 *
+	 * @return string|array Value of setting selected by $key or full array of settings.
+	 */
+	public static function get_settings( $key = NULL ) {
+		$options = get_site_option( 'wpme_options' );
+		$options = wp_parse_args( $options, self::$default_options );
+
+		if ( isset( $key ) )
+			return $options[ $key ];
+		else
```

Comment:
The `else` is not necessary.

Reply:
Ha! Good catch! :) Also fixed in https://github.com/bueltge/wordpress-multisite-enhancements/pull/63/commits/62b2e77f7c438c952204639cc4193ab4e64914d0